src/FeedForwardPid.cpp

```diff
@@ -6,27 +6,45 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+#include <limits>
+
 #include "iarc7_motion/FeedForwardPid.hpp"
 
 using namespace Iarc7Motion;
 
-FeedForwardPid::FeedForwardPid(double p_gain, double i_gain, double d_gain) : p_gain_(p_gain), i_gain_(i_gain), d_gain_(d_gain), setpoint_(0.0)
+FeedForwardPid::FeedForwardPid(double p_gain, double i_gain, double d_gain,
+                               double i_accumulator_max, double i_accumulator_min)
+    : p_gain_(p_gain),
+      i_gain_(i_gain),
+      d_gain_(d_gain),
+      setpoint_(0.0),
+      i_accumulator_(0.0),
+      last_current_value_(0.0),
+      i_accumulator_max_(i_accumulator_max),
+      i_accumulator_min_(i_accumulator_min)
 {
     // Nothing to do
 }
 
 double FeedForwardPid::update(double current_value, double time_delta)
 {
-
-    static double i_accumulator{0.0};
-    static double last_current_value{0.0};
-
+    if (time_delta == 0) {
+        time_delta = std::numeric_limits<double>::min();
```

Comment:
Also since we are taking the time I think we should properly handle this. Not just use min. It works for this applications output scale but would not always be a good thing.